augur/application/db/data_parse.py

```diff
@@ -466,8 +466,22 @@ def extract_needed_contributor_data(contributor, tool_source, tool_version, data
 
     return contributor
 
+def extract_needed_clone_history_data(clone_history_data:List[dict], repo_id:int):
```

Comment:
@IsaacMilarky / @ABrain7710 : What needs fixing here?

Reply:
This function LGTM unless it's throwing errors.